up.sh

```diff
@@ -4,6 +4,15 @@
 # The `-s` flag takes a string of services to run.
 # The `-l` flag will use mounted code.
 
+VERSION=$(docker-compose --version)
+node -e "
```

Comment:
We should remove the dependency on node, this should be possible to do with `sed` or `awk`